### DISCUSSION TITLE
Verifying MX records?

### DISCUSSION THREAD

**Discussion by cryo75** [MichaCo/DnsClient.NET#197] [General]
Domains like `icoud.com` or `googglemail.com` return answers but they should be incorrect mail exchanges. How can I make sure that the domain is a valid mail exchange?

**Comment by MichaConrad** [comment]
I do not understand what you mean by, you got an answer and they should be incorrect?!

And I have no idea how to properly validate an email server. But you could probably just use the smpt protocol / or a mail client , and try to connect to see what the response is? 
Anyways, that has nothing todo with DNS and I'm not really sure what you are asking for.

**Reply by funkel1989** [reply]
@cryo75 those domains return answers because someone or some entity owns them or are forwarding messages from those domains to a real domain with a dns server that has been setup with an mxrecord. The dns client package is not wrong.
